source/data/data_management_guidelines.rst

```diff
@@ -0,0 +1,159 @@
+
+A few more things to keep in mind:
+
+- When developing software with multiple people, it may sound tempting to place the code on shared storage for this purpose. A much better practice, however, is to stick to personal copies in combination with a suitable :ref:`version control system <version control systems>`. Know that ICTS offers `a yearly course on version control <https://admin.kuleuven.be/icts/opleidingen/opleidingsaanbod/version-control-hybrid>`_ and that KU Leuven also hosts `an own GitLab instance <https://gitlab.kuleuven.be/>`_.
+
+- Conda environments tend to take up a lot of disk space as well as inodes (number of files and directories). Inode counts can be strongly reduced by installing such environments in a :ref:`container <hpc containers>`.
+
+- Carefully consider where to place your software installations in your directory structure. Ask yourself e.g. whether or not the software needs to be available to the whole group and whether it will be used for a single research project or across several ones. In general, software installation directories are best well separated from other types of data in your project.
+
+
+Data outflow to long-term storage
+=================================
+
+Two more points are worth considering with regards to outflow: (1) selecting which data to transfer and (2) which transfer tools to use.
+
+Which data needs to be moved to external storage?
+-------------------------------------------------
+Your HPC storage may contain a variety of data (publicly available datasets, measurements from your instruments, generated output data, intermediate files, et cetera). While you could just move everything to your external storage, we recommend to only transfer the data which **really** needs to be preserved after the active computation phase. Aside from reducing your storage demands and associated costs, this also improves searchability. This triage is best done before your outflow activities (transferring everything first and cleaning up later usually means the cleanup does not get done). If data meets any of the following criteria, then it is likely to be a good candidate for outflow:
+
+- Data which is expensive to regenerate in terms of compute time, transfer time and/or human effort.
+- Data which needs to be shared with people without access to VSC facilities, such as external partners in a research project or the general public (for example through :ref:`Globus <globus-sharing>` or `KU Leuven RDR <https://www.kuleuven.be/rdm/en/rdr>`_).
+
```

Comment:
'Note that you do not need to wait until data is processed to put it on external storage. 
You could consider copying raw data early to an external location, to safeguard it or to share it with others, while keeping a copy on scratch or staging for processing'.